Metadata/AmazonMetadataBuilder.php

```diff
@@ -114,6 +115,18 @@ protected function getDefaultMetadata()
      */
     protected function getContentType($filename)
     {
+        // NEXT_MAJOR: Remove this BC hack
```

Comment:
WDYT @sonata-project/contributors ? Could we drop Guzzle and Buzz completly without any deprecation notes on the current stable release?